src/screens/Classes.js

```diff
@@ -19,6 +20,7 @@ class Classes extends Component {
     return (
       <ScreenContainer screenTitle='Classes'>
         <TextBody bodyColor={theme.COLOR_BLACK}>Classes!</TextBody>
+        <TeamButton>Create Team</TeamButton>
```

Comment:
Instead of a hard-coded string, please use props so that we can reuse the component if need be!

Reply:
updated!